content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/used.rb

```diff
@@ -6,7 +6,7 @@ def consumption(source)
   {
     :cpu                 => source.allocated_vcpu,
     :memory              => source.allocated_memory,
-    :vms                 => source.vms.count { |vm| vm.id unless vm.archived },
+    :vms                 => source.vms.count { |vm| vm.id unless vm.archived || vm.orphaned },
```

Comment:
@branic @tinaafitz Instead of checking each method would it be valid to just check `unless vm.ems_id.nil?`.  (Or even better `if vm.ems_id`)

That is the common condition between the two methods:
https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template.rb#L1269-L1279